### PR TITLE
Linking to new LS docs and New to Omics

### PR DESCRIPTION
**Pull request by pm0kjp**
@leemc-data-ed and @allrolsen and @nafeldman can you take a peek at the changes here?  I'm linking to New to Omics and the LS documents from the main Arcus Lab Orientation.

### CODE REVIEW

**Comment by allrolsen** [comment]
> @leemc-data-ed and @allrolsen and @nafeldman can you take a peek at the changes here? I'm linking to New to Omics and the LS documents from the main Arcus Lab Orientation.

Thank you for adding links to our LS modules. I think the copy looks good!

File: arcus_orientation.md [hunk 1]
```diff
@@ -78,11 +79,24 @@ You should begin by working in this training document, which constitutes an Arcu
...
+- **[New to Omics](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/main/new_to_omics.md#1)** will get you started if you're using the Arcus Omics Training Lab.  Note that this document is best used specifically within our Omics Training Lab, where we have pre-defined data that you can work with.  The instructions in this document presuppose that these files exist and that you're working in the Omics Training Lab.
```
Comment (leemc-data-ed) [inline arcus_orientation.md:99]: The wording here seems a bit repetitive, maybe something like this? 
```suggestion
- **[New to Omics](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/main/new_to_omics.md#1)** will get you started if you're using the Arcus Omics Training Lab. The instructions in this document presuppose that specific files, containing pre-defined data, exist and that you're working in the Omics Training Lab.
```

**Review by leemc-data-ed — CHANGES_REQUESTED** [review]
Just one small suggestion, but otherwise this looks good to me!

**Comment by nafeldman** [comment]
looks awesome, thank you both!

**Review by leemc-data-ed — APPROVED** [review]
Lgtm!

**Pull request merged**